ext/release-lead.rake

```diff
@@ -0,0 +1,139 @@
+require 'json'
+require 'optparse'
+require 'yaml'
+
+namespace :release_lead do
+  desc "Find platforms added and removed between releases"
+  task :platform_diff, [:from, :to] do |t, args|
+    abort ('`from` argument is required') unless args.from
+    abort ('`to` argument is required') unless args.to
+    puts "Platform Changes:"
+    puts
+    puts `git diff --summary #{args.from}..#{args.to} configs/platforms`.
+        gsub('configs/platforms/', '').gsub(/mode \d+ /, '').
+        gsub('delete', 'Retired').gsub('create', 'Added').
+        gsub('rename', 'Renamed')
+  end
+
+  desc "List currently supported platforms on the current branch"
+  task :supported_platforms do |t, args|
```

Comment:
I like this addition; does it make sense to reference this on the [Confluence page](https://confluence.puppetlabs.com/display/systemsEngineering/Puppet+Agent+Release+Lead+Guide)?

Reply:
Yeah, it likely does. We need to mention on the confluence page that these tasks are not exact: sometimes names can look similar enough git thinks it's a rename when it's both an addition + removal.

Additionally supported_platforms won't list the platforms where we build one package for multiple supported platforms (AIX, Windows, el 7 (we use the 7 package for AWS) etc.)